### ISSUE TITLE
Integration test for 0.1% Zurich

### ISSUE THREAD

**Issue by sebhoerl** [matsim-eth/baseline_scenario#35]
We need an integration test for the scenario synthesis.

Two options:
- Either we move this repository to ETH gitlab, then we can run integration tests on pikelot or ifalik (where we also have the raw data)
- Or we create some "dummy" microcensus that is shipped with this repository (probably the better solution)

**Comment by tduberne** [comment]
I would also favor the fake data option.

But if running on our own servers ends up being easier, rather than a full move to gitlab, one could setup a mirror. Gitlab provides such functionality, but it seems to be limited to Gitlab EE (and ETH uses CE, and a rather old version of it): https://docs.gitlab.com/ee/workflow/repository_mirroring.html

Or one could setup a cron job on the ETH servers that pull from GitHub eg. every Saturday, runs the tests and sends the results to Slack.

**Comment by sebhoerl** [comment]
Yes, I think only the fake case makes sense, working on it. I don't like the polling option, because I want a direct feedback for PRs. Nevertheless, it will to set up a cron job for building the "big" scenarios.

**Comment by tduberne** [comment]
> Yes, I think only the fake case makes sense, working on it

:+1: 

> I don't like the polling option, because I want a direct feedback for PRs

True, did not think in pull requests. The mirror option would also fail to provide that functionality, so full move to Gitlab would probably be the best option if we need to run the tests on our servers
